Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Parameters/SpeckleBlockInstanceWrapper.ModelObjects.cs

```diff
@@ -186,13 +177,11 @@ private bool CreateFromInstanceReference(InstanceReferenceGeometry instanceRef)
       Base = new InstanceProxy()
       {
         definitionId = definitionId.ToString(),
-        maxDepth = 1,
+        maxDepth = 0, // represent newly created, top-level objects. actual depth calculation happens in GrasshopperBlockPacker
         transform = GrasshopperHelpers.TransformToMatrix(instanceRef.Xform, units),
-        units = units,
-        applicationId = Guid.NewGuid().ToString()
+        units = units
```

Comment:
in object wrapper, all application id assignements were happening on cast. Reasoning for moving this out?